Sources/SwerverCore/Router/Router.swift

```diff
@@ -12,10 +12,11 @@ public class Router {
         populateRoutes()
     }
 
-    public func process(request: HttpRequest) -> String {
+    public func process(request: HttpRequest?) -> String {
 
-        let targetURL = request.getUrl()
-        let targetMethod = request.getMethod()
+        guard let targetURL = request?.getUrl(), let targetMethod = request?.getMethod() else {
```

Comment:
Typically, it would be conventional to unwrap the `request` itself once, and not for each attribute on the request.

e.g.
```swift
func route(request: String?) -> String {
    if let request = request {
        return "Valid Request: \(request)"
    } else {
        return "Invalid Request"
    }
}

print(route(request: "Good")) // "Valid Request: Good"
print(route(request: nil)) // "Invalid Request
```

Reply:
will fix that